### PR TITLE
Issue #11446: Update CheckTest to use verifyWithInlineConfigParser

### PR DESCRIPTION
**Pull request by chrccl**
Issue #11446: Update CheckTest to use verifyWithInlineConfigParser

### CODE REVIEW

**Comment by chrccl** [comment]
Good morning... I didn't understand why, when I made the pull request for issue #11446, it also included yesterday's commit from issue #13213. I wanted to pull only 2 commits (one where I modified the file, and the other I made just because I forgot to update the pom.xml). 
@romani  can you help me to fix the failing checks?

File: src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java [hunk 1]
```diff
@@ -580,35 +580,27 @@ public void testDestroyCheckerWithWrongCacheFileNameLength() throws Exception {
...
-        checkerConfig.addProperty("charset", StandardCharsets.UTF_8.name());
-
         final File cacheFile = File.createTempFile("junit", null, temporaryFolder);
-        checkerConfig.addProperty("cacheFile", cacheFile.getPath());
 
         final File tmpFile = File.createTempFile("file", ".java", temporaryFolder);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
 
-        verify(checkerConfig, tmpFile.getPath(), expected);
+        verifyWithInlineConfigParser(tmpFile.getPath(), expected);
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java:596]: You can not simply change method, you need to add to input file check that was added for execution in Junit. Please review other changes that were done for this issue to get more examples

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java (hunk 1 above)
Comment (chrccl) [reply src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java:596]: perfect, thanks so much for the suggestions, I'll get to it straight away

**Comment by romani** [comment]
@chrccl , please do not close PRs . Reuse them.

**Pull request closed without merging**